src/fileCreator/transforms.ts

```diff
@@ -71,3 +78,20 @@ function escapeRegExp(str): string {
 function replaceAll(str, find, replace): string {
   return str.replace(new RegExp(escapeRegExp(find), "g"), replace);
 }
+
+function replaceDynamicOptions(content: string, dynamicOptions: string): string {
+
+  var result = content;
+
+  const options = dynamicOptions.split(";");
+
+  for (var i = 0; i < options.length; i++) {
+
+      const opt = options[i];
+      const handle = "{{$" + (i + 1) + "}}";
```

Comment:
I think we want to be more lenient on this replacement... right now `{{ $1 }}` would not match due to the spacing. Instead of doing this search ourselves can we use handlebars to do the replacement?